examples/thermomechanics/inputs/thermal_deformation.json

```diff
@@ -1,8 +1,9 @@
 {
     "num_cells"             : {"value": [100, 30, 3]},
     "system_size"           : {"value": [1.0, 0.3, 0.03], "unit": "m"},
+    "num_types"             : {"value": 1},
     "density"               : {"value": 3980,  "unit": "kg/m^3"},
-    "elastic_modulus"       : {"value": 370e+9, "unit": "Pa"},
+    "elastic_modulus"       : {"value": [370e+9, 100e+9, 10e+9], "unit": "Pa", "note": "11, 22, 12"},
```

Comment:
Why do we have 3 values here? I believe we only need for a bimaterial the elastic modulus of material 1 and material 2.

Reply:
I moved this to draft. But until we have a bi-material example, I need places to test things are working